src/library_browser.js

```diff
@@ -335,18 +335,22 @@ var LibraryBrowser = {
           canvas.exitFullscreen = canvas.exitFullscreen.bind(document);
           if (Browser.lockPointer) canvas.requestPointerLock();
           Browser.isFullscreen = true;
-          if (Browser.resizeCanvas) Browser.setFullscreenCanvasSize();
+          if (Browser.resizeCanvas)
```

Comment:
need `{`, `}` around if and else bodies

Reply:
Done, but I'm wondering if it's just a style issue or a syntax one?